gfx-examples/src/shadow.rs

```diff
@@ -421,7 +443,7 @@ impl framework::Example for Example {
             let uniform_size = mem::size_of::<ShadowUniforms>() as u32;
             let uniform_buf = device.create_buffer(&wgpu::BufferDescriptor {
                 size: uniform_size,
-                usage: wgpu::BufferUsageFlags::UNIFORM | wgpu::BufferUsageFlags::TRANSFER_DST,
+                usage: wgpu::BufferUsageFlags::UNIFORM | wgpu::BufferUsageFlags::TRANSFER_DST | wgpu::BufferUsageFlags::MAP_WRITE,
```

Comment:
I think many of those `TRANSFER_DST` flags aren't needed anymore. They were only required by `set_sub_data`